pkg/cmd/openshift-kube-apiserver/server.go

```diff
@@ -0,0 +1,83 @@
+package openshift_kube_apiserver
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/glog"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	utilwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/pkg/version"
+	aggregatorinstall "k8s.io/kube-aggregator/pkg/apis/apiregistration/install"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/kubernetes/pkg/capabilities"
+	kubelettypes "k8s.io/kubernetes/pkg/kubelet/types"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+	"github.com/openshift/origin/pkg/cmd/server/apis/config/validation"
+	"github.com/openshift/origin/pkg/cmd/server/origin"
+	"github.com/openshift/origin/pkg/cmd/util/variable"
+)
+
+func RunOpenShiftKubeAPIServerServer(masterConfig *configapi.MasterConfig) error {
+	// Allow privileged containers
+	capabilities.Initialize(capabilities.Capabilities{
```

Comment:
this sounds like something I would expect in admission, and not that early in the flow.

Reply:
> this sounds like something I would expect in admission, and not that early in the flow.

This actually appears indicative of kubelet flags leaking across processes.  And yeah, it may belong in admission long term, but for now we need it on the process and it affects static validation code.